packages/web/src/components/table/Table.tsx

```diff
@@ -201,7 +201,7 @@ export const Table = ({
     }
   }, [columns, defaultSorter, isVirtualized, onSort, sortBy, useLocalSort])
 
-  useEffect(handleSortChange, [sortBy])
```

Comment:
this may cause issues with how many times the handleSortChange function is called. There was a warning here, but the intent was so only that the handleSortChange function called then sortBy was changed, not the function itself. Would love to discuss how to better fix this

Reply:
Ohhh nice yeah good call, I'll just add an ignore comment then